.circleci/config.yml

```diff
@@ -2,11 +2,36 @@ version: 2
 jobs:
   make-eksctl-image:
     # use machine build, as otherwise we cannot use make and have to duplicate makefile
-    machine: true
+    machine:
+      image: ubuntu-1604:201903-01
+    environment:
+        GO_VERSION: 1.12.6
+        GOCACHE: /home/circleci/.cache/go-build/
+        GOPATH: /home/circleci/go
     steps:
       - checkout
+      - restore_cache:
+            keys:
+                - go-cache-{{ .Branch }}-{{ .Revision }}
+                - go-cache-{{ .Branch }}-
+                - go-cache-
+      - run:
+            name: Install Golang (needed for dependency calculation)
+            command: |
+                curl -OL https://storage.googleapis.com/golang/go${GO_VERSION}.linux-amd64.tar.gz
```

Comment:
Really?

Reply:
Ehm, yes? I don't know what you mean by that. Please elaborate.

Reply:
Ah, you edited the comment.

Reply:
It's only used for the dependency computation in the makefile. Nothing to do with construction or the like.

If you want I can factor out `make eksctl-image`  in a script and avoid installing it altogether.

Reply:
Maybe, but don't have to do it now. Can we just cache it, or install it with `apt`?

Reply:
It takes 5 seconds to install so it's not worth caching. Apt in Ubuntu 16 doesn't have a newer version and I am not sure it's worth the pain of installing another APT repo.

Reply:
Yeah, I see! Anyway, let's leave it as is, the tarball is hosted in GCP, which is quite fast and reliable. I'd hope we will find a nice CI solution one day ;)